packages/block-library/babel-plugin.js

```diff
@@ -0,0 +1,279 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+
+/**
+ * Internal dependencies
+ */
+const isBlockMetadataExperimental = require( './src/is-block-metadata-experimental' );
+
+const BLOCK_LIBRARY_INDEX_PATH = 'block-library/src/index.js';
```

Comment:
Is it possible, with Babel config, to enable this plugin just for the one affected source file?

Reply:
That was my question too! Unfortunately I didn't find such an option in the documentation: https://github.com/jamiebuilds/babel-handbook/blob/master/translations/en/plugin-handbook.md

Reply:
That's documentation for plugin writers, but this is more of a config file thing. Something like this in `babel.config.js` could work:
```js
{
  overrides: {
    test: 'packages/block-library/src/index.js',
    plugins: [ 'babel-plugin-inline-json-import' ],
  }
}
```
[Here](https://babeljs.io/docs/en/configuration#pluginpreset-merging) they say that the `plugins` array will be merged into the default one.